tests/test__database_api.py

```diff
@@ -18,7 +18,11 @@ def test__get_collection_by_attribute_underscore(self):
         database['_users'].insert_one({'a': 1})
         self.assertEqual(1, database['_users'].find_one().get('a'))
 
+    def test__command_ping(self):
```

Comment:
Just a suggestion, you could maybe add an additional test case to ensure that command as a string works:
```suggestion
    def test__command_ping_as_str(self):
        database = mongomock.MongoClient().somedb
        self.assertEqual({'ok': 1}, database.command('ping'))

    def test__command_ping(self):
```

Reply:
Woops, you're right. I've also added a test for a string containing "ping" but not "ping".